libs/ui/src/env.d.ts

```diff
@@ -1,7 +1,6 @@
 declare namespace NodeJS {
   export interface ProcessEnv {
-    readonly NODE_ENV: 'development' | 'production' | 'test';
-    readonly REACT_APP_VX_DEV?: string;
-    readonly REACT_APP_VX_DISABLE_CARD_READER_CHECK?: string;
+    NODE_ENV: 'development' | 'production' | 'test';
+    REACT_APP_VX_SKIP_PIN_ENTRY?: string;
```

Comment:
I realize you're removing `readonly` for testing purposes, but I'm not sure that's the right tradeoff. I think in other tests we've chosen to mock the feature flag function, not `process.env` directly.

@eventualbuddha what do you think?

Reply:
FWIW, I chose to mock it this way because [in other tests we have mocked `process.env` directly](https://github.com/votingworks/vxsuite/blob/2b7cbb1eb674cdf9b8e902cf6560761dfacbe5df/libs/utils/src/environment_flag.test.ts#L30).

Reply:
I think I agree with @benadida here. We're changing `process.env` directly in that test because the code under test is the thing that accesses `process.env`. There are some other places we do it, but we also have places where we're [mocking the `isFeatureFlagEnabled` function](https://github.com/votingworks/vxsuite/blob/7b6a705f25164e145e9153df84e20c2be29f208c/frontends/precinct-scanner/src/screens/scan_warning_screen.test.tsx#L81-L85). I think neither of these approaches are great, really. I think I'd prefer something that can be controlled with a React context that we could use like this:

```ts
// in a component
const isSkipPinEntryEnabled = useFeatureFlag(EnvironmentFlagName.SKIP_PIN_ENTRY);

useEffect(() => {
  if (isSkipPinEntryEnabled) {
    auth.checkPasscode(auth.user.passcode);
  }

// …

// in a test
renderInAppContext(
  <SomeComponent />,
  {
    featureFlags: {
      [EnvironmentFlagName.SKIP_PIN_ENTRY]: true
    }
  }
);
```

But we don't need to do that in this PR. I'd prefer changing this to do the mocking as I linked to above, but I don't think it's required.

Reply:
I switched to the other way to mock.